tests/integration/test_task_pool.py

```diff
@@ -62,7 +63,7 @@
 def get_task_ids(
     name_point_list: Iterable[Tuple[str, Union[PointBase, str, int]]]
 ) -> List[str]:
-    """Helper function to return sorted task identities ("{name}.{point}")
+    """Helper function to return sorted task identities
     from a list of  (name, point) tuples."""
```

Comment:
I actually think that the original form was helpful. How about:?

```suggestion
    """Helper function to return sorted task identities
    from a list of  (name, point) tuples.
    
    Returns: name.point
    """
```

Reply:
It was out of date since universal ID. Do we really need an explanation of IDs every time they are used?

Reply:
No.